consensus/core/src/dag_state.rs

```diff
@@ -175,8 +235,7 @@ impl DagState {
     }
 
     /// Gets all ancestors in the history of a block at a certain round.
-    /// The round must be higher than the last committed round.
-    pub(crate) fn ancestors_at_uncommitted_round(
+    pub(crate) fn ancestors_at_round(
```

Comment:
Since this method now seems to be more generic, shall we still keep the check at line 243? It's nice sanity check but nothing a the method's signature points to such restriction.